src/core/server/app/handlers/api/oEmbed/oEmbed.ts

```diff
@@ -0,0 +1,64 @@
+import { createFetch } from "coral-server/services/fetch";
+import { RequestHandler } from "coral-server/types/express";
+
+const fetchURL = (url: string, type: string) => {
+  if (type === "twitter") {
+    return `https://publish.twitter.com/oembed?url=${url}`;
+  }
+  if (type === "youtube") {
+    return `https://www.youtube.com/oembed?url=${url}`;
+  }
+
+  return "";
+};
+
+export const oEmbedHandler = (): RequestHandler => {
```

Comment:
Handlers should be reflective of their mounting points. Coral enforces a strict lowercase on URL's, this should be renamed to:

```suggestion
export const oembedHandler = (): RequestHandler => {
```